oraclecloud-function-http/src/main/java/io/micronaut/oraclecloud/function/http/FnMultiValueMap.java

```diff
@@ -39,14 +41,27 @@
  */
 @Internal
 public class FnMultiValueMap implements ConvertibleMultiValues<String> {
+    protected ConversionService conversionService;
     private final Map<String, List<String>> map;
 
     /**
      * Default constructor.
-     * @param map The target map. Never null
+     *
+     * @param map               The target map. Never null
      */
     public FnMultiValueMap(Map<String, List<String>> map) {
+        this(map, new DefaultMutableConversionService());
```

Comment:
I think this should be

```suggestion
        this(map, ConversionService.SHARED);
```